taigaProject/src/taigaApi/task/getTaskHistory.py

```diff
@@ -68,7 +68,7 @@ def get_task_lead_time(project_id, auth_token):
             "taskId": task["id"],
             "startTime": task["created_date"],
             "endTime": task['finished_date'],
-            "timeTaken": lead_time
+            "timeTaken": (finished_date - created_date).days
```

Comment:
Earlier we were adding the difference to leadTime. We just need the difference between start and finish time for a particular task.